### PR TITLE
Fixed small functionality issues

### LINKED ISSUE #44: Sort Logs By Date
Logs should be sorted by date (desc order)

### LINKED ISSUE #46: Parse exercise from string

### LINKED ISSUE #51: Calorie count feature
1. restricting the exercises available (situp, pushup) add in json
2. calculate calorie in log 
3. display calorie

### PR DESCRIPTION
**Pull request by WhiteLio**

### CODE REVIEW

File: src/main/java/seedu/fma/ui/PersonCard.java [hunk 1]
```diff
@@ -34,6 +34,8 @@
     @FXML
     private Label reps;
     @FXML
+    private Label cal;
```
Comment (therizhao) [inline src/main/java/seedu/fma/ui/PersonCard.java]: Could rename to calories for clarity

File: src/main/java/seedu/fma/model/ModelManager.java [hunk 2]
```diff
@@ -36,7 +37,9 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
 
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
-        filteredLogs = new FilteredList<>(this.addressBook.getPersonList());
+        // TODO: Sorting order is currently hardcoded
```
Comment (therizhao) [inline src/main/java/seedu/fma/model/ModelManager.java:39]: What do u mean by hardcoded here?

**Review by therizhao — CHANGES_REQUESTED** [review]
minor fixes

File: src/main/java/seedu/fma/model/ModelManager.java (hunk 2 above)
Comment (WhiteLio) [reply src/main/java/seedu/fma/model/ModelManager.java:39]: its the next line where I sort with a lambda, ideally if we implement sorting in the future we want to properly use SortedList and Comparators instead

File: src/main/java/seedu/fma/ui/PersonCard.java (hunk 1 above)
Comment (WhiteLio) [reply src/main/java/seedu/fma/ui/PersonCard.java]: aight will do

File: src/main/java/seedu/fma/model/ModelManager.java (hunk 2 above)
Comment (WhiteLio) [reply src/main/java/seedu/fma/model/ModelManager.java:39]: no need to worry about it now haha

**Review by therizhao — APPROVED** [review]
lgtm

**Pull request merged**